src/main/java/com/iota/iri/conf/DbConfig.java

```diff
@@ -55,32 +55,32 @@
     boolean isRescanDb();
 
     /**
-     * Default Value: {@value BaseIotaConfig.Defaults#TX_BATCH_WRITE}
+     * Default Value: {@value BaseIotaConfig.Defaults#TX_CACHE_SIZE}
      *
-     * @return {@value DbConfig.Descriptions#TX_BATCH_WRITE}
+     * @return {@value DbConfig.Descriptions#TX_CACHE_SIZE}
      */
-    long getTxBatchWrite();
+    long getTxCacheSize();
```

Comment:
small peev can this be an int?
Don't see why we would ever need such a large cache

Reply:
ok